setup.cfg

```diff
@@ -1,5 +1,72 @@
+############################################################################
+# Copyright(c) Open Law Library. All rights reserved.                      #
+# See ThirdPartyNotices.txt in the project root for additional notices.    #
+#                                                                          #
+# Licensed under the Apache License, Version 2.0 (the "License")           #
+# you may not use this file except in compliance with the License.         #
+# You may obtain a copy of the License at                                  #
+#                                                                          #
+#     http: // www.apache.org/licenses/LICENSE-2.0                         #
+#                                                                          #
+# Unless required by applicable law or agreed to in writing, software      #
+# distributed under the License is distributed on an "AS IS" BASIS,        #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. #
+# See the License for the specific language governing permissions and      #
+# limitations under the License.                                           #
+############################################################################
+
+[metadata]
+name = pygls
+version = 0.9.1
+author = Open Law Library
+author_email = info@openlawlib.org
+license = Apache 2.0
+description = a pythonic generic language server (pronounced like "pie glass").
+keywords = python, pythonic, generic, language, server, protocol
+url = https://github.com/openlawlibrary/pygls/tree/master/
+long_description = file: README.md
+long_description_content_type = text/markdown
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Topic :: Software Development :: Libraries :: Python Modules
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+packages = find:
+zip_safe = False
+install_requires = pydantic==1.7; typeguard==2.10.0
```

Comment:
From `setup.py` -> `install_requires=["pydantic>=1.7,<1.9", "typeguard>=2.10.0,<3", "psutil>=5"],`.

Reply:
You can also write each dependency on the new line, for better readability.

Reply:
Fixed.